examples/llm/throughput/README.md

```diff
@@ -63,8 +63,12 @@ python submit_benchmarks.py -m 13b.yaml 30b.yaml -t fp16 -b 21 21 -s 11 14 --RUN
 ```
 This will run 8 configs for 12 steps to get throughput numbers. `python collect_results.py` can then be used to parse all output training logs and create the tables below.
 
+Sometimes we report Global Batchsizes that are not evenly divisible by the micro batch size. Our microbatching engine allows for non-divisible sizes, while being mathematically faithful to the global batch size. For example, a total batch size of 48, and a micro batch of 11, means we will accumulate gradients across microbatches of 11, 11, 11, 11, 4. 
```

Comment:
Should the warning at the top of this file be removed yet?

Reply:
Probably not yet... the 13B/30B/70B models in some of the rows are still old. Maybe the warning should be reworded slightly though?

Reply:
Updated the warning a bit.